cscs-checks/microbenchmarks/osu/osu_tests.py

```diff
@@ -114,23 +111,21 @@ def setup(self, partition, environ, **job_opts):
 
```

Comment:
The `setup()` method here obfuscates a bit the code. It does a setup for GPU benchmarks practically and the base class is supposed to be agnostic. I suggest moving this code in the constructor of the corresponding GPU benchmark subclasses, where you will only check against the `self.current_system` and not against the partition. Also for Daint/Dom you should load `craype-accel-nvidia60` instead of `cudatoolkit`.